README.md

```diff
@@ -40,8 +42,9 @@ LET VERSION = 1.5.0
 
 4. Upload data: This step is tied to the implementation of Cell-Feature-Explorer. The current strategy is to upload two large directories of files: the OME-TIFF files, and the postprocessed json and png files.
    To upload the json and png data:
+   `set DATASET_VERSION=1.5.0`
    `cd %prefs.out_dir%`
-   `s3promotion upload --dest-bucket bisque.allencell.org --dest-prefix v%VERSION%/ %dataset_constants.THUMBNAILS_DIR%/`
+   `s3promotion upload --dest-bucket bisque.allencell.org --dest-prefix v%DATASET_VERSION%/ %dataset_constants.THUMBNAILS_DIR%/`
```

Comment:
nit: Why not make this a single unified code block?

Reply:
That is a todo.  bin/deploy_release.py will be a separate script that is not yet written.